index.html

```diff
@@ -0,0 +1,63 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>Awesome Books ES6</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="#">Awesome Books With ES6</a>
+
+            <ul id="nav-list">
+                <li><a data-main="1" href="#">List</a></li> |
+                <li><a data-main="2" href="#">Add new</a></li> |
+                <li><a data-main="3" href="#">Contact</a></li>
+            </ul>
+
+        </nav>
+        <p id="date"></p>
+        
+    </header>
+
+    <section class="main main1">
+        <h1>All awesome books</h1>
+
+        <div id="new-books"></div>
+    </section>
+
+    <section class="main main2 display">
+        <h2>Add a new book</h2>
+
+        <form action="">
+            <input id="book-title" placeholder="Title" type="text">
+            <input id="book-author" placeholder="Author" type="text">
```

Comment:
- [ ] **Optional:** Attempt to clear the input field after entering a book. Also, try to restrict multiple entries of the same book and author in your spare time.